Loop/View Controllers/BolusViewController.swift

```diff
@@ -319,7 +367,12 @@ final class BolusViewController: ChartsTableViewController, IdentifiableClass, U
 
     private func updateDeliverButtonState() {
         let deliverText = NSLocalizedString("Deliver", comment: "The button text to initiate a bolus")
-        if potentialCarbEntry == nil {
+
+        if isLoggingDose {
+            footerView.primaryButton.setTitle(NSLocalizedString("Log Dose", comment: "The button text to log an insulin dose not given by the pump"), for: .normal)
+            footerView.primaryButton.isEnabled = enteredBolusAmount != nil && enteredBolusAmount! > 0
+            footerView.primaryButton.tintColor = enteredBolusAmount != nil && enteredBolusAmount! > 0 ? .alternateBlue : .systemBlue
```

Comment:
Why prefer `alternateBlue` as the action button color?

Reply:
No good reason, removed.